### PR TITLE
[ADD] web_menu_navbar_needaction

### PR DESCRIPTION
**Pull request by hbrunn**
# Needaction counters in main menu

This module shows the sum of all submenus' needaction counters in main menus. This way, users get visual feedback where there's still work to do without having to open the respective menu beforehand.

The counters are updated periodically, so users will also be notified if there are new records that required attention.
# Configuration

To configure the update frequency, set the configuration parameter `web_menu_navbar_needaction.refresh_timeout` to the amount of milliseconds after which the counters should be updated. Don't do this too often, as this will cause a lot of queries to the database. The default it 600000, which is equivalent to 10 minutes.
# Usage

.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
    :target: https://runbot.odoo-community.org/runbot/162/8.0

### CODE REVIEW

**Comment by rafaelbn** [comment]
Tested in runbot :+1:

**Comment by nhomar** [comment]
:+1:

File: web_menu_navbar_needaction/README.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+Needaction counters in main menu
```
Comment (pedrobaeza) [inline web_menu_navbar_needaction/README.rst:4]: Put upper line

File: web_menu_navbar_needaction/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Needaction counters in main menu",
+    "version": "1.0.0",
```
Comment (pedrobaeza) [inline web_menu_navbar_needaction/__openerp__.py]: Use 8.0.1.0.0

**Comment by pedrobaeza** [comment]
:+1: functional test. It works like a charm

Some minor remarks though for the README and manifest

**Comment by hbrunn** [comment]
@pedrobaeza thanks, done

**Comment by guewen** [comment]
:+1:

**Pull request merged**